packages/manager/src/features/linodes/LinodesCreate/SelectPlanPanel/PlanInformation.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { LinodeTypeClass } from '@linode/api-v4/lib/linodes';
+import Typography from 'src/components/core/Typography';
+import { useSelectPlanPanelStyles } from './styles/plansPanelStyles';
+import { planTabInfoContent } from './utils';
+import { GPUNotice } from './GPUNotice';
+import { MetalNotice } from './MetalNotice';
+import { PremiumPlansAvailabilityNotice } from '../PremiumPlansAvailabilityNotice';
+
+interface Props {
+  disabledClasses?: LinodeTypeClass[];
+  planType: LinodeTypeClass;
+}
+
+export const PlanInformation = ({ disabledClasses, planType }: Props) => {
+  const { classes } = useSelectPlanPanelStyles();
+
+  const getDisabledClass = (thisClass: string) => {
+    const inactiveClasses = (disabledClasses as string[]) ?? []; // Not a big fan of the casting here but it works
+    return inactiveClasses.includes(thisClass);
+  };
```

Comment:
What about something like this instead?

```suggestion
  const getDisabledClass = (thisClass: LinodeTypeClass) => {
    return Boolean(disabledClasses?.includes(thisClass));
  };
```